src/labkey/Pipeline.ts

```diff
@@ -96,25 +88,23 @@ export interface IGetPipelineContainerOptions {
      * been defined in this container hierarchy, the value of this property will be null.
      * - webDavURL: the WebDavURL for the pipeline root.
      */
-    success?: Function
+    success?: RequestSuccess
```

Comment:
Same comment as above, any reason this success can't be removed and come from RequestCallbackOptions?

Reply:
See comment above.